frontend/src/component/common/FeatureArchiveDialog/FeatureArchiveDialog.test.tsx

```diff
@@ -144,6 +153,11 @@ test('Show error message when multiple parents of orphaned children are archived
     await screen.findByText(
         'have child features that depend on them and are not part of the archive operation. These parent features can not be archived:',
     );
+    expect(
```

Comment:
we should not show the warning if there's already an error about orphaned children. Error and warning would confuse people